tap-snapshots/test/lib/commands/publish.js.test.cjs

```diff
@@ -310,14 +310,6 @@ exports[`test/lib/commands/publish.js TAP public access > new package version 1`
 + @npm/test-package@1.0.0
 `
 
-exports[`test/lib/commands/publish.js TAP re-loads publishConfig.registry if added during script process > new package version 1`] = `
```

Comment:
Not sure how these got removed for you locally but you'll need to revert this file for tests to pass.